bin/Makefile.base

```diff
@@ -15,6 +15,12 @@ test_fmt:
 	out=`gofmt -l -s .`; echo "$$out"; [ -z "$$out" ]
 	go vet ./...
 
+.PHONY: test_imports
+test_imports:
+	@echo Checking correct formatting and import lines of files
+	out=`goimports -l .`; echo "$$out"; [ -z "$$out" ]
+	go vet ./...
```

Comment:
It's already used in `test_fmt`, consider spliting the `test_fmt` into a "top level" phony rule depending on smth like `test_go-vet`, `test_go-fmt` & `test_go-imports`, or merge the two recipes.

Reply:
Yes I would eventually want to do this but adding `goimports` into the `test_fmt` would be would be a "breaking" change for the repos that are currently using it. I'm not too aware of repos that would be affected.
I agree it would be best but I though there can be a first step of just make the rule `test_imports` available as a replacement for `go_fmt`. It might be best to have both for now without `go vet` in `test_imports`

Reply:
Ho, right, that would break some repos, good catch!